### ISSUE TITLE
Zen mode shortcut overlaps with the "edit code" shortcut on windows

### ISSUE THREAD

**Issue by thakkarparth007** [getcursor/cursor#623]
This is a minor issue.

Zen mode shortcut: Ctrl+K Z
Edit code shortcut: Ctrl+K

<img width="430" alt="image" src="https://github.com/getcursor/cursor/assets/747423/e28023bf-4a34-4d4c-b810-1216ce21f23a">

I entered zen mode by the view menu, that hid the menubar, and couldn't use keyboard shortcut to exit the zen mode.

**Comment by thakkarparth007** [comment]
I realized there are some other overlaps too. The all-too-familiar Ctrl+D now maps to aichat.autodebugterminal.

Ctrl+shift+e doesn't seem to work although I don't know what's wrong in the keyboard shortcuts settings
<img width="690" alt="image" src="https://github.com/getcursor/cursor/assets/747423/db088e81-c798-4575-9723-1f2bd2c8432e">

**Comment by truell20** [comment]
Just to be clear, were you able to fix these collisions by using the shortcut editor?

**Comment by thakkarparth007** [comment]
Yes, but not the ctrl+shift+e one. Not sure why that's not working.

**Comment by photomz** [comment]
Same thing for `Ctrl+K C` and `Ctrl+K U` for comment/uncomment keyboard bindings!
